extensions/data-transfer/portability-data-transfer-google/src/test/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItemTest.java

```diff
@@ -84,81 +86,101 @@ public void getMimeType_videoModel_mimeTypeFromFilename() {
-
-    assertEquals("image/webp", photoModel.getMimeType());
+      assertEquals("image/webp", photoModel.getMimeType());
   }
 
   @Test
-  public void getMimeType_videoModel_filenameMimeTypeIsNull() {
+  public void getMimeType_videoModel_filenameMimeTypeIsNull() throws ParseException{
     GoogleMediaItem videoMediaItem = getVideoMediaItem();
     videoMediaItem.setFilename("file");
     videoMediaItem.setMimeType("video/webm");
+      VideoModel videoModel = GoogleMediaItem.convertToVideoModel(Optional.empty(), videoMediaItem);
 
-    VideoModel videoModel = GoogleMediaItem.convertToVideoModel(Optional.empty(), videoMediaItem);
-
-    assertEquals("video/webm", videoModel.getMimeType());
+      assertEquals("video/webm", videoModel.getMimeType());
   }
 
   @Test
-  public void getMimeType_photoModel_nullMimeTypeReturnsDefault() {
+  public void getMimeType_photoModel_nullMimeTypeReturnsDefault() throws ParseException{
     GoogleMediaItem photoMediaItem = getPhotoMediaItem();
     photoMediaItem.setFilename("file");
     photoMediaItem.setMimeType(null);
-
     PhotoModel photoModel = GoogleMediaItem.convertToPhotoModel(Optional.empty(), photoMediaItem);
 
     // Default defined in GoogleMediaItem
     assertEquals("image/jpg", photoModel.getMimeType());
   }
 
   @Test
-  public void getMimeType_videoModel_nullMimeTypeReturnsDefault() {
+  public void getMimeType_videoModel_nullMimeTypeReturnsDefault() throws ParseException{
     GoogleMediaItem videoMediaItem = getVideoMediaItem();
     videoMediaItem.setFilename("file");
     videoMediaItem.setMimeType(null);
-
     VideoModel videoModel = GoogleMediaItem.convertToVideoModel(Optional.empty(), videoMediaItem);
 
     // Default defined in GoogleMediaItem
     assertEquals("video/mp4", videoModel.getMimeType());
   }
 
   @Test
-  public void getMimeType_photoModel_unsupportedFileExtension() {
+  public void getMimeType_photoModel_unsupportedFileExtension() throws ParseException{
     GoogleMediaItem photoMediaItem = getPhotoMediaItem();
     photoMediaItem.setFilename("file.avif");
     photoMediaItem.setMimeType("image/png");
-
     PhotoModel photoModel = GoogleMediaItem.convertToPhotoModel(Optional.empty(), photoMediaItem);
 
     // defaults to the mimetype that is already set, as avif is not handled.
     assertEquals("image/png", photoModel.getMimeType());
   }
 
+  @Test
+  public void getUploadTime_videoModel() throws ParseException{
+    GoogleMediaItem videoMediaItem = getVideoMediaItem();
+    MediaMetadata metadata = new MediaMetadata();
+    metadata.setVideo(new Video());
+    metadata.setCreationTime("2023-10-02T22:33:38Z");
+    videoMediaItem.setMediaMetadata(metadata);
+    VideoModel videoModel = GoogleMediaItem.convertToVideoModel(Optional.empty(), videoMediaItem);
+
+    assertEquals("Mon Oct 02 22:33:38 UTC 2023", videoModel.getUploadedTime().toString());
+
+  }
+
+  @Test
+  public void getUploadTime_photoModel() throws ParseException{
```

Comment:
are we testing uploadTime or creationTime in these two tests? The name of the test says uploadTime, but then we set the creationTime and then assert on the uploadTime. Maybe a few comments and renaming the test would be good here

Reply:
I have already added the comments to the model specifying what creation time is vs upload time. Things were named badly. I don't think we should be doing the entire name refactoring in this cl so I have commented it as a TODO in the models.

Reply:
I more mean for us to add a comment to this test, since it's hard to understand what the test is doing